src/components/postsList/container/postsListContainer.tsx

```diff
@@ -56,29 +58,50 @@ const postsListContainer = ({
       
     }, [scrollPosition])
 
+
+    const _setImageHeightInMap = (postId:number, height:number) => {
```

Comment:
shouldn't postId be `string` now?

Reply:
agreed, it should have been string before as well.. thanks